gslb/ingestion/gslb.go

```diff
@@ -670,6 +662,16 @@ func Initialize() {
 		gslbutils.LogAndPanic("error building kubernetes clientset: " + err.Error())
 	}
 
+	amkoControlConfig := gslbutils.AMKOControlConfig()
+	amkoControlConfig.SetClientset(kubeClient)
+
+	pod, err := kubeClient.CoreV1().Pods(gslbutils.AVISystem).Get(context.TODO(), os.Getenv("POD_NAME"), metav1.GetOptions{})
+	if err != nil {
+		utils.AviLog.Warnf("Error getting AMKO pod details.")
+	}
+	amkoControlConfig.SaveAMKOPodObjectMeta(pod)
```

Comment:
Would recommend to use `pod.DeepCopy()` here instead of using a pointer to the pod directly.